static_src/components/dropdown.jsx

```diff
@@ -0,0 +1,29 @@
+
+import React from 'react';
```

Comment:
Yea. It also makes it easier for future development. The hope with component design is that you don't have to write code every time new interface features get added, you can piece together what you've already build.